### PR TITLE
PREAPPS-3042: Email folder sharing | Update Notification Emails UI

### PR DESCRIPTION
**Pull request by labderahul1**

### CODE REVIEW

File: src/batch-client/index.ts [hunk 1]
```diff
@@ -595,7 +595,56 @@ export class ZimbraBatchClient {
...
+								parentFolderAbsPath,
+								folder.folders
+							);
+						}
+
+						return folder;
+					});
+
+				folders.linkedFolders = folders.linkedFolders.map((folder: any) => {
+					if (folder.view === FolderView.Message) {
```
Comment (silentsakky) [inline src/batch-client/index.ts]: don't we need this for other message types?

Comment (labderahul1) [reply src/batch-client/index.ts]: branch 3042 is created from 2589. This code is added in 2589.
This code is not added by me.

**Comment by silentsakky** [comment]
Closing this PR as there are no changes in this branch respect to base branch PREAPPS-2589

File: src/batch-client/index.ts (hunk 1 above)
Comment (SantoshKShetty) [reply src/batch-client/index.ts]: @silentsakky Folders will have message/contact/appointment views. Are there any more ?

Comment (silentsakky) [reply src/batch-client/index.ts]: below is the list of valid values from api docs
`|search folder|tag|conversation|message|contact|document|appointment|virtual conversation|remote folder|wiki|task|chat`

**Pull request closed without merging**